CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ All notable changes to this project will be documented in this file.
   separate documents.
 - Updated user interface with loading icon to provide visual representation of
   the request's duration.
+- Changed the homepage to update when a file is selected not when it is analysed, based on client feedback.
+- Increased the response timeout delay from 5 seconds to 15 seconds.
```

Comment:
i don't think this should be here, it's more of an implementation detail than a user facing change